.gitignore

```diff
@@ -2,4 +2,7 @@ __pycache__/
 *.py[cod]
 *.log
 app.cfg
+appbackup.cfg
+venv_bot_p37
+*swo
```

Comment:
Those additions probably don't hurt, but they are somewhat specific to your environment. For now this is fine, however it might be better to separate code and runtime environment better. Maybe you could make a new issue  _Separate bot code from runtime environment_ Part of the runtime environment could be log files, `app.cfg`, virtual environments (`venv*`).